src/components/discover/ClearAllButton.tsx

```diff
@@ -10,7 +11,7 @@ export const ClearAllButton: React.FC<ClearAllButtonProps> = ({ className }) =>
   const clearAll = useDiscoverQueryStore((state) => state.resetToDefaultInitState)
 
   return (
-    <Button onClick={clearAll} variant='secondary' size='md' className={className}>
+    <Button onClick={clearAll} variant='secondary' size='md' className={cn('h-10 min-w-[100px]', className)}>
```

Comment:
you should not update height
It should be 42px per figma if I'm not mistaken. That's a standard height for button that has normal size